### PR TITLE
Added:  Date Picker view in the events section 

### LINKED ISSUE #627: Date Picker view in the events section
**Is your feature request related to a problem? Please describe.**
Currently we have filter functionalities in the event section which are now not working for the calendar, we can replace it with a live small DatePicker.

**Describe the solution you'd like**
A small date picker like in general calendars
**Actual behaviour**
Screenshot 2023-03-10 at 12 54 30 AM (2)

**Expected behaviour**
Screenshot 2023-03-10 at 12 18 16 AM (2)

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by aashimawadhwa**
**What kind of change does this PR introduce?**
feature

**Issue Number:**
Fixes #627 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**
Screenshot 2023-03-30 at 7 07 03 AM (2)

**If relevant, did you update the documentation?**
Not required

**Summary**
Implemented date picker and removed the old filters that worked for event tiles.

**Does this PR introduce a breaking change?**
no

**Other information**
not any

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
yes

### CODE REVIEW

**Review by ttaylor92 — CHANGES_REQUESTED** [review]
Please cover the line marked by code coverage as we do not want to decrease the overall coverage percentage currently existing.

**Comment by EshaanAgg** [comment]
@aashimawadhwa You have simply removed the code for functionality that allows to filter by `Title`, `Description` and `Location`. Is there a specific reason for it?

**Comment by aashimawadhwa** [comment]
@EshaanAgg The calendar for talawa is inspired from google calendar. if you look closely to the calendar  we have search functionality and a small calendar in the sidebar, even at breeze chms we have the styling the same way.

Therefore we are removing the filter functionality for now

**Comment by palisadoes** [comment]
Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

1) The overall code coverage drops below the target threshold of the repository
2) Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained. This is a prerequisite for getting your PR approved. Assigned reviewers regularly review the PR queue and tend to focus on PRs that are passing.

**Comment by Rumone** [comment]
> @EshaanAgg The calendar for talawa is inspired from google calendar. If you look closely to the calendar we have a small calendar in the sidebar, even at breeze chms we have this in the same way.
> 
> 
> 
> Therefore we are removing the filter functionality for now and adding the small calendar.
I don't see the benefit of removing the existing filter functionality. Would they not provide valuable results to the user? The mini calendar could augment the existing filter 
**Example** Filter by a specific title and a date.

**Comment by aashimawadhwa** [comment]
@Rumone  the location and description is not a properly defined therefore I don't think it will be a good parameter to search for user, and this calendar implemented  is not for users specifically it is focused on the super admins of the organization as of now.

**Comment by EshaanAgg** [comment]
@aashimawadhwa Earlier, the `Description` and `Location` fields were also shown on the dashboard; thus, they made good candidates to filter events. Since now only the `Title` field is shown on the UI, at least the support to filter by the same should be provided and not deprecated.

The date picker can be used in two ways:
1. To change the focus of the calendar (which month or day I am exploring)
2. As a filter to sort by date (and date ranges)
What do you think would be a better functionality from a long-term prospect?

File: src/screens/OrganizationEvents/OrganizationEvents.tsx [hunk 1]
```diff
@@ -138,75 +137,19 @@ function OrganizationEvents(): JSX.Element {
...
-  /* istanbul ignore next */
-
-  const handleSearchByTitle = (e: any) => {
-    const { value } = e.target;
-    const filterData = {
-      organization_id: currentUrl,
-      title_contains: value,
-    };
-    refetch(filterData);
-  };
```
Comment (EshaanAgg) [inline src/screens/OrganizationEvents/OrganizationEvents.tsx:150]: As you can see here, the filtering indeed does work (via an API call to the backend).

**Comment by palisadoes** [comment]
Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

1) The overall code coverage drops below the target threshold of the repository
2) Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained. This is a prerequisite for getting your PR approved. Assigned reviewers regularly review the PR queue and tend to focus on PRs that are passing.

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by aashimawadhwa** [comment]
Sir, I am making  some changes, will update this ASAP. @palisadoes

**Comment by aashimawadhwa** [comment]
@palisadoes Sir, can you close this PR, I will raise a new one. This is having a lot of conflicts which I am unable to solve.

**Pull request closed without merging**